lib/desired-caps.js

```diff
@@ -75,6 +75,9 @@ let desiredCapConstraints = _.defaults({
   wdaStartupRetryInterval: {
     isNumber: true
   },
+  prebuildWDA: {
```

Comment:
I guess there could be. But I also like the symmetry of the two. I'm hoping that this is temporary and only necessary on certain systems.